src/test/java/Api/DataForPojo.java

```diff
@@ -0,0 +1,386 @@
+package Api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import jdk.nashorn.internal.objects.annotations.Getter;
+import jdk.nashorn.internal.objects.annotations.Setter;
+
+@JsonDeserialize(as = DataForPojo.class)
+public class DataForPojo {
```

Comment:
pojo class should be located in resorces/dto directory (despite we placed it here on the lesson)

Reply:
Please name this class more informative. This class is POJO